### PR TITLE
non-rpas & base station reqs added

### PR DESCRIPTION
**Pull request by wzfftxwd**

### CODE REVIEW

File: docs/reqs/reqs.tex [hunk 1]
```diff
@@ -217,7 +217,11 @@ \subsection{Multirotor RPAS}
 
 \subsection{Non-RPAS Air-Ground I/O}
 \begin{enumerate}[label=F.CM.\arabic*, wide=1cm, widest=3cm, leftmargin=*, font=\bfseries, noitemsep,topsep=0pt, parsep=4pt, partopsep=0pt]
-	\item Computation platform transmits video data and ML model data to ground station.
+	\item The bi-directional communication between base station and computing platfrom shall have a transmission distance of at least 30 meters.
+	\item The communication shall be established automatically.
+	\item The communication shall be estabilished under commonly-used protocols and does not require proprietary equipment.
```
Comment (peterdeutsch) [inline docs/reqs/reqs.tex]: Commonly-used protocols is vague here

File: docs/reqs/reqs.tex [hunk 2]
```diff
@@ -217,7 +217,11 @@ \subsection{Multirotor RPAS}
...
+	\item Computation platform shall transmit video data and ML model data to ground station.
+	\item Base station shall send commands to the computing platform.
```
Comment (peterdeutsch) [inline docs/reqs/reqs.tex]: What commands?

File: docs/reqs/reqs.tex [hunk 3]
```diff
@@ -234,8 +238,13 @@ \subsection{Machine Learning Implementation}
...
 \begin{enumerate}[label=F.BS.\arabic*, wide=1cm, widest=3cm, leftmargin=*, font=\bfseries, noitemsep,topsep=0pt, parsep=4pt, partopsep=0pt]
-    \item Ground station displays the video data overlaid with the ML model data.
-	\item ML model data is presented in the form of bounding boxes.
+    \item Base station shall display the video data overlaid with the ML model data.
+	\item Base station shall display the ML model data in the form of bounding boxes.
+	\item Base station shall support bi-directional data transfer with computing platform.
+	\item Base station shall have a user interface for sending control commands to the computing platform.
+    \item Base station shall be capable of storing metadata received from the drone for further analysis.
+    \item Base station shall be in the form of a mobile, battery powered device.
+    \item Base station shall relay system faults/warnings to the user.
```
[the remark this answers is not in the document]
Comment (wzfftxwd) [reply docs/reqs/reqs.tex]: Should we list out all possibles ones or just provide an example?

[the remark this answers is not in the document]
Comment (peterdeutsch) [reply docs/reqs/reqs.tex]: Hmm, maybe we could change it to "Base station shall prominently display system faults/warnings to the user."

File: docs/reqs/reqs.tex [hunk 4]
```diff
@@ -217,11 +217,10 @@ \subsection{Multirotor RPAS}
 
 \subsection{Non-RPAS Air-Ground I/O}
 \begin{enumerate}[label=F.CM.\arabic*, wide=1cm, widest=3cm, leftmargin=*, font=\bfseries, noitemsep,topsep=0pt, parsep=4pt, partopsep=0pt]
-	\item The bi-directional communication between base station and computing platfrom shall have a transmission distance of at least 30 meters.
-	\item The communication shall be established automatically.
-	\item The communication shall be estabilished under commonly-used protocols and does not require proprietary equipment.
-	\item Computation platform shall transmit video data and ML model data to ground station.
-	\item Base station shall send commands to the computing platform.
+	\item The Air-Ground communication between base station and computing platfrom shall have a transmission distance of at least 30 meters.
+	\item The Air-Ground communication shall be estabilished under IEEE 802.11n protocal.
```
Comment (peterdeutsch) [inline docs/reqs/reqs.tex]: *protocol

File: docs/reqs/reqs.tex [hunk 5]
```diff
@@ -217,11 +217,10 @@ \subsection{Multirotor RPAS}
...
+	\item The Air-Ground communication shall be established automatically.
+	\item The Air-Ground communication shall be bi-directional.
```
Comment (peterdeutsch) [inline docs/reqs/reqs.tex]: We can combine this with the first point.

**Pull request merged**